crawler_mcp/core/streaming.py

```
@@ -0,0 +1,369 @@
+
+    async def stop(self) -> None:
+        """Stop processing."""
+        self._processing = False
+        
+        if self._processor_task:
+            self._processor_task.cancel()
+            try:
+                await self._processor_task
+            except asyncio.CancelledError:
+                pass
+
```

Ruff (0.12.2)

Use `contextlib.suppress(asyncio.CancelledError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(asyncio.CancelledError)`

(SIM105)